src/WebJobs.Script.Abstractions/Description/Binding/BindingMetadata.cs

```diff
@@ -16,6 +17,14 @@ public class BindingMetadata
     {
         private const string _systemReturnParameterBindingName = "$return";
 
+        public BindingMetadata() : this(new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase)) { }
```

Comment:
My code suggestion had sloppy (lazy) formatting. Not sure if that matters to you much, but I would format this as:
```suggestion
        public BindingMetadata()
            : this(new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase))
        {
        }
```